static/js/disease/data_processing_utils.ts

```diff
@@ -2,8 +2,69 @@ import _ from "lodash";
 
 import { GraphNodes } from "../shared/types";
 import { DiseaseGeneAssociationData } from "./chart";
+/**
```

Comment:
please add the license header to new files.

Reply:
Done!